src/renderer/components/dialogs/EditProfileDialog/ProfileImageSelector.tsx

```diff
@@ -1,8 +1,11 @@
 import React from 'react'
 
 import ImageSelector from '../../ImageSelector'
+import ImageCropper from '../../ImageCropper'
```

Comment:
Can we also add this cropper for editing group images?

Reply:
I think I've dragged this one for a while :) So maybe I'll make a new PR for the group images?